events/views.py

```diff
@@ -45,3 +48,17 @@ def export_to_csv(request):
             error = "Please Enter a Valid Token"
             return render(request, 'download.html', {'error': error})
     return render(request, 'download.html')
+    
+def requestEvent(request):
+    if request.method == 'POST':
+        form = requestEventForm(request.POST)
+        if form.is_valid():
+            form.save()
+            your_name = form.cleaned_data.get('your_name')
+            title = form.cleaned_data.get('title')
+            messages.success(
+                request, f'Thank you { your_name } for requesting to organise { title }, you will be contacted shortly!')
+            return redirect('homepage')
+    else:
+        form = requestEventForm()
+    return render(request, 'event_register.html', {'form': form})
```

Comment:
While creating methods, you should not write like this.
Use this notation, `def request_event(request):`